Description: Fix spelling error in the mysql plugin
 This patch fix a spelling error in the mysql plugin
Author: Gennaro Oliva <oliva.g@na.icar.cnr.it>
Forwarded: https://bugs.schedmd.com/show_bug.cgi?id=15950
Last-Update: 2022-12-21

--- a/src/plugins/accounting_storage/mysql/as_mysql_assoc.c
+++ b/src/plugins/accounting_storage/mysql/as_mysql_assoc.c
@@ -528,7 +528,7 @@
 	}
 	xfree(query);
 	if (!(row = mysql_fetch_row(result))) {
-		debug4("Can't move a none existent association");
+		debug4("Can't move a non-existent association");
 		mysql_free_result(result);
 		return ESLURM_INVALID_PARENT_ACCOUNT;
 	}
--- a/NEWS
+++ b/NEWS
@@ -9297,7 +9297,7 @@
  -- Set the number of free licenses to be 0 if the global license count
     decreases and total is less than in use.
  -- Add DebugFlag of BackfillMap. Previously a DebugFlag value of Backfill
-    logged information about what it was doing plus a map of expected resouce
+    logged information about what it was doing plus a map of expected resource
     use in the future. Now that very verbose resource use map is only logged
     with a DebugFlag value of BackfillMap
  -- Fix slurmstepd core dump.
--- a/src/common/env.c
+++ b/src/common/env.c
@@ -1999,7 +1999,7 @@
 		/* skip any env variables with a newline in newline mode */
 		if (newline && xstrstr(*p, "\n")) {
 			log_flag_hex(STEPS, *p, strlen(*p),
-				     "%s: skiping environment variable with newline",
+				     "%s: skipping environment variable with newline",
 				     __func__);
 			continue;
 		}
--- a/src/common/slurm_errno.c
+++ b/src/common/slurm_errno.c
@@ -580,7 +580,7 @@
 	{ ERRTAB_ENTRY(ESLURM_PLUGIN_INCOMPLETE),
 	  "Plugin missing required symbol or function"		},
 	{ ERRTAB_ENTRY(ESLURM_PLUGIN_NOT_LOADED),
-	  "Required plugin type not loaded or initalized"	},
+	  "Required plugin type not loaded or initialized"	},
 
 	/* REST errors */
 	{ ERRTAB_ENTRY(ESLURM_REST_INVALID_QUERY),
--- a/src/slurmctld/job_mgr.c
+++ b/src/slurmctld/job_mgr.c
@@ -7196,7 +7196,7 @@
 		}
 		if ((job_desc->time_min) && (job_desc->time_min != NO_VAL) &&
 		    (job_desc->deadline < (earliest_start + min_in_sec))) {
-			info("%s: job's min_time excedes the deadline (%s + %lu > %s)",
+			info("%s: job's min_time exceeds the deadline (%s + %lu > %s)",
 			     __func__, time_str_earliest, min_in_sec,
 			     time_str_deadline);
 			rc = ESLURM_INVALID_TIME_MIN_LIMIT;
@@ -7206,7 +7206,7 @@
 		    (job_desc->time_limit) &&
 		    (job_desc->time_limit != NO_VAL) &&
 		    (job_desc->deadline < (earliest_start + limit_in_sec))) {
-			info("%s: job's time_limit excedes the deadline (%s + %lu > %s)",
+			info("%s: job's time_limit exceeds the deadline (%s + %lu > %s)",
 			     __func__, time_str_earliest, limit_in_sec,
 			     time_str_deadline);
 			rc = ESLURM_INVALID_TIME_LIMIT;
--- a/doc/man/man5/burst_buffer.conf.5
+++ b/doc/man/man5/burst_buffer.conf.5
@@ -62,7 +62,7 @@
 .TP
 \fBDirective\fR
 The string that must be used by a job to request a burst buffer. This string
-must be immediately preceeded by a single '#' character. This is currently only
+must be immediately preceded by a single '#' character. This is currently only
 used by the lua plugin. For the lua plugin, the default value is "BB_LUA".
 See burst_buffer.html for more details.
 .IP
--- a/doc/man/man1/scrun.1
+++ b/doc/man/man1/scrun.1
@@ -179,7 +179,7 @@
 
 .TP
 \fBSCRUN_STDERR_DEBUG\fR=<quiet|fatal|error|info|verbose|debug|debug2|debug3|debug4|debug5>
-Set logging level for standard error ouput only.
+Set logging level for standard error output only.
 .IP
 
 .TP
--- a/doc/man/man5/slurm.conf.5
+++ b/doc/man/man5/slurm.conf.5
@@ -7737,8 +7737,8 @@
 .br
 2. Modify the slurm.conf file appropriately.
 .br
-3. Copy the files from the StateSaveLocation to the new contoller or ensure
-that they are accessible to the new contoller via a shared drive.
+3. Copy the files from the StateSaveLocation to the new controller or ensure
+that they are accessible to the new controller via a shared drive.
 .br
 4. Distribute the updated slurm.conf file to all nodes.
 .br
